### PR TITLE
data/selinux: merge SELinux policy module

### PR DESCRIPTION
**Pull request by Conan-Kudo**
This PR merges the SELinux policy module I had been developing on [my repository on GitLab](https://gitlab.com/Conan_Kudo/snapcore-selinux) into the snapd repository.

Hopefully now, SELinux policy development will now be part of snapd development.

### CODE REVIEW

File: data/selinux/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# snapd-selinux
+
+This is the SELinux policy for Snapd and related Snappy components
+
+# Licensing
+
+As the work from this is derived from modules from [Fedora's SELinux policy project](https://github.com/fedora-selinux/selinux-policy), it is licensed in the same manner.
```
Comment (zyga) [inline data/selinux/README.md]: Could you please re-format this to (vim select + gq) to wrap nicely?

File: data/selinux/COPYING [hunk 2]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (zyga) [inline data/selinux/COPYING]: I wonder if this should go to top-level as COPYING.GPL2, @mvo5 what do you think.

We should be clear about the license that applies to particular part of the code.

@Conan-Kudo I think we need to document the license in the particular files to be compliant.

Comment (Conan-Kudo) [inline data/selinux/COPYING]: Done.

**Comment by mvo5** [comment]
This looks good to me, thank you! Really excited to see SELinux landing :-D

**Comment by Conan-Kudo** [comment]
🎉

**Pull request merged**